File: src/basic/sum/js/sum_1.js
/**
 * n명의 국어 점수 중 80점 이상의 총점 구하기
 */

//[1] Input : n명의 국어 점수
let scores = [100, 75, 50, 37, 90, 95];
let sum = 0;

//[2] Process
for (let i = 0; i < scores.length; i++) {
	if (scores[i] >= 80) {
		sum += scores[i];
	}
}

//[3] Output
console.log(scores.length + "명의 점수 중 80점 이상의 총점은 : " + sum);

// RUN
// eclipse - JAVA >  Ctrl + F11
// vscode - Code Runner > Ctrl + Alt + N

File: src/basic/sum/Sum1.java
package basic.sum;

/**
 * n명의 국어 점수 중 80점 이상의 총점 구하기
 */
public class Sum1 {

	public static int[] scores = { 100, 75, 50, 37, 90, 95 };

	public static void main(String[] args) {
		int sum = 0;

		for (int score : scores) {
			if (score >= 80) {
				sum += score;
			}
		}
		System.out.println(scores.length + "명의 점수 중 80점 이상의 총점은 : " + sum);
	}
}
